template.yaml

```diff
@@ -473,6 +473,36 @@ Resources:
             Path: /person/{id}/employment/{employmentId}/
             Method: patch
 
+  NvaCristinListFundingSourcesFunction:
+    Description: Function for listing all funding sources
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: cristin-funding
+      Handler: no.unit.nva.cristin.funding.sources.query.ListFundingSourcesHandler::handleRequest
+      Role: !GetAtt ReadCristinAuthenticationRole.Arn
```

Comment:
You dont need this for an open endpoint

Reply:
It is what allows function to add auth header when doing authorized requests to cristin

Reply:
I will delete.